modules/check_in/app/workers/check_in/travel_claim_submission_worker.rb

```diff
@@ -22,9 +22,15 @@ class TravelClaimSubmissionWorker
     STATSD_BTSSS_ERROR = 'worker.checkin.travel_claim.btsss.error'
     STATSD_BTSSS_DUPLICATE = 'worker.checkin.travel_claim.btsss.duplicate'
 
+    PATIENT_CELL_PHONE_FEATURE = Flipper.enabled?('check_in_experience_patient_cell_phone') || false
+
     def perform(uuid, appointment_date)
       check_in_session = CheckIn::V2::Session.build(data: { uuid: })
-      mobile_phone = TravelClaim::RedisClient.build.mobile_phone(uuid:)
+      mobile_phone = if PATIENT_CELL_PHONE_FEATURE
```

Comment:
I think this might be better:

```suggestion
      mobile_phone = if Flipper.enabled?(:check_in_experience_patient_cell_phone)
```